modules/packages/LinearAlgebra.chpl

```diff
@@ -1787,7 +1795,7 @@ proc _norm(x: [?D], param p: normType) where x.rank == 2 {
   }
 }
 
-/* Return the solution ``x`` to the linear system `` L * x = b ``
+/* Return the solution ``x`` to the linear system ``L * x = b``
     where ``L`` is a lower triangular matrix. Setting `unit_diag` to true
```

Comment:
This seems to be rendering oddly, though I'm not sure why what you've done would have impacted it.  I think it may help to either shift the first line in by one space, or all the following lines in the comment back by one space